### PR TITLE
Indigo publish fixes

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: .github/workflows/indigo-ci.yaml [hunk 1]
```diff
@@ -705,18 +705,39 @@ jobs:
...
+      - name: Download Python wheels
+        uses: actions/download-artifact@v2
+        with:
+          name: indigo-python
+          path: dist/
+      - name: Download Java jars
+        uses: actions/download-artifact@v2
+        with:
+          name: indigo-java
+          path: dist/
```
Comment (mkviatkovskii) [inline .github/workflows/indigo-ci.yaml:723]: We don't need to download jars, only native libraries

File: .github/workflows/indigo-ci.yaml [hunk 2]
```diff
@@ -705,18 +705,39 @@ jobs:
...
+          name: indigo-dotnet
           path: dist/
-      - name: Update Wrappers version
-        run: python3 build_scripts/indigo-update-version.py
       - name: Publish Python wheels to PyPI
         env:
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
-        run: python3 build_scripts/indigo-make-by-libs.py --type=python --wrappers-arch=universal --publish
+        run: |
+             ls dist/
```
Comment (mkviatkovskii) [inline .github/workflows/indigo-ci.yaml:733]: Let's remove all debug prints

File: .github/workflows/indigo-ci.yaml [hunk 3]
```diff
@@ -725,15 +746,14 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.MK_GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.MK_GPG_PASSPHRASE }}
         run: |
+          pwd
+          ls -alh dist/
```
Comment (mkviatkovskii) [inline .github/workflows/indigo-ci.yaml:750]: These debug prints too.

File: .github/workflows/indigo-ci.yaml (hunk 1 above)
Comment (even1024) [reply .github/workflows/indigo-ci.yaml]: fixed

File: .github/workflows/indigo-ci.yaml (hunk 2 above)
Comment (even1024) [reply .github/workflows/indigo-ci.yaml]: fixed

File: .github/workflows/indigo-ci.yaml (hunk 3 above)
Comment (even1024) [reply .github/workflows/indigo-ci.yaml]: fixed

File: .github/workflows/indigo-ci.yaml [hunk 4]
```diff
@@ -725,15 +740,13 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.MK_GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.MK_GPG_PASSPHRASE }}
         run: |
+          pwd
```
Comment (mkviatkovskii) [inline .github/workflows/indigo-ci.yaml:743]: We don't need pwd either

**Pull request merged**